### PR TITLE
Re-submit "Enable IPv6 Private peering config on express route circuit connection"

### PR DESCRIPTION
**Pull request by anton-evseev**
We created `network-january-branch` with incorrect API version and had to remove it to re-create. But one PR was already merged into it: https://github.com/Azure/azure-rest-api-specs/pull/8200

This PR is to retake that one into new branch. Also, I think ARM should also review this.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [X] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by number213** [comment]
@anthony-c-martin could you please review this from ARM's side?

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json [hunk 1]
```diff
@@ -1948,6 +1948,19 @@
...
     },
+    "Ipv6CircuitConnectionConfig": {
+      "description": "IPv6 Circuit Connection properties for Global Reach.",
+      "properties": {
+        "addressPrefix": {
+          "type": "string",
+          "description": "/125 IP address space to carve out Customer addresses for Global Reach."
+        },
+        "circuitConnectionStatus": {
+          "$ref": "#/definitions/CircuitConnectionStatus",
```
Comment (anthony-c-martin) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json]: Is this something the user can update or is it a read-only property? If read-only, please mark it as such.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json [hunk 2]
```diff
@@ -1948,6 +1948,19 @@
         "modelAsString": true
       }
     },
+    "Ipv6CircuitConnectionConfig": {
+      "description": "IPv6 Circuit Connection properties for Global Reach.",
```
Comment (anthony-c-martin) [inline specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json]: Elsewhere it's written `global reach` (no caps). Please could you make this consistent?

**Review by anthony-c-martin — CHANGES_REQUESTED** [review]
See comments

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json (hunk 1 above)
Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json]: Marked as read-only

File: specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json (hunk 2 above)
Comment (number213) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/expressRouteCircuit.json]: Done

**Comment by number213** [comment]
@anthony-c-martin addressed your comments, PTAL

**Review by pilor — APPROVED** [review]

**Comment by number213** [comment]
@allenjzhang ARM signed off, could you please merge this PR?

**Comment by number213** [comment]
@allenjzhang SDK Net keeps hitting timeout. Otherwise CI looks good. Could you please merge?

**Comment by allenjzhang** [comment]
> @allenjzhang SDK Net keeps hitting timeout. Otherwise CI looks good. Could you please merge?

The .NET issue can be ignored. Though the merge is blocked because Anthony's review is still on Change Requested. Once that is signed off, then I can do the merge.

**Comment by number213** [comment]
@anthony-c-martin could you please approve this PR or reset your review?

**Review by anthony-c-martin — APPROVED** [review]

**Comment by number213** [comment]
@allenjzhang please merge

**Pull request merged**